content/spire/getting-started.md

```diff
@@ -62,40 +62,38 @@ The key manager generates and persists the public-private key pair used for the
 
 * Which trust root (“upstream certificate authority (CA)”) your application will use
 
-The SPIRE server provides a CA. If you’re, for example, using an external PKI system
-	that provides an upstream CA. you can configure SPIRE to use that instead.
+The SPIRE server provides a CA. If you’re, for example, using an external PKI system that provides an upstream CA, you can configure SPIRE to use that instead.
 
-Once you’ve made these decisions, you can [configure the server](#step-4-configure-the-server) and [configure the agent](#step-5-configure-the-agent) accordingly, after [installing them](#step-3-install-the-server-and-agent). 
+Once you’ve made these decisions, you can [configure the server](#step-4) and [configure the agent](#step-5) accordingly, after [installing them](#step-3-install-the-server-and-agent). 
 
-## Step 2 Obtain the SPIRE Binaries
+## Step 2: Obtain the SPIRE Binaries {#step-2}
 
-Pre-built SPIRE releases can be found in the [GitHub SPIRE releases directory](https://github.com/spiffe/spire/releases). These releases contain both server and agent binaries.
+Pre-built SPIRE releases can be found on the [SPIRE downloads page](/downloads#spire). These releases contain both server and agent binaries.
 
 If you wish, you may also [build SPIRE from source](https://github.com/spiffe/spire/blob/master/CONTRIBUTING.md).
 
-## Step 3 Install the Server and Agent
+## Step 3: Install the Server and Agent {#step-3}
 
 As stated above, this guide illustrates installs the server and agent on the same node. More typically, your architecture will have the the server installed on one node and one or more agents installed on distinct nodes. 
 
 To install the server and agent:
 
-1. Obtain the latest tarball from [the GitHub SPIRE releases directory](https://github.com/spiffe/spire/releases): 
+1. Obtain the latest tarball from [the SPIRE downloads page](/downloads#spire) and then extract it into the **/opt/spire** directory using the following commands:
 
-and then extract it into the **/opt/spire** directory using the following commands:
+	```shell
+	wget https://github.com/spiffe/spire/releases/download/{{< spire-latest >}}/spire-{{< spire-latest >}}-linux-x86_64-glibc.tar.gz
+	sudo tar zvxf spire-{{< spire-latest >}}-linux-x86_64-glibc.tar.gz
```

Comment:
You'll notice here that injecting `{{< spire-latest >}}` can automatically infer the latest version of SPIRE (which means that you don't need to hardcode that version).

Reply:
LGTM.